package.json

```diff
@@ -60,17 +60,18 @@
         "cli-table3": "^0.6.0",
         "conf": "^6.2.4",
         "googleapis": "^49.0.0",
-        "inquirer": "^7.1.0",
+        "inquirer": "^7.3.3",
         "inquirer-datepicker-prompt": "^0.4.2",
         "keytar": "^6.0.1",
-        "mongoose": "^5.9.11",
+        "mongoose": "^5.10.15",
         "mysql": "^2.18.1",
-        "pg": "^8.5.1",
         "node-cron": "^2.0.3",
-        "nodemailer": "^6.4.6",
-        "omelette": "^0.4.12",
-        "ora": "^4.0.4",
-        "ssh2-sftp-client": "^5.1.2"
+        "node-mailer": "^0.1.1",
```

Comment:
Can you check this dependency. It does not look like the package we are using for nodemailer.

Reply:
sure.